examples/config.yaml

```diff
@@ -21,20 +21,11 @@ tools:
       name: go
       version: "1.17"
     branch: main
-    jobs:
-      build:
```

Comment:
if “build” doesn’t exist, we enable it by default; if “build command” doesn’t exist, we set it as “go build ./..." as default.

---

I think `build` needs to be retained, otherwise users will not know how to fill in when they need to specify the build command. The corresponding code logic behind may also need to be adjusted accordingly